### ISSUE TITLE
Workspace - Not here page is briefly displayed when selecting a workspace

### ISSUE THREAD

**Issue by lanitochka17** [Expensify/App#40999]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 1.4.66-0
**Reproducible in staging?:** Y
**Reproducible in production?:** Y
**If this was caught during regression testing, add the test name, ID and link from TestRail:** N/A
**Email or phone of affected tester (no customers):** natnael.expensify+3@gmail.com
**Issue reported by:** Applause - Internal Team

## Action Performed:
1. Sing up with a new account
2. Create manual expense in Self DM
3. Create workspace
4. Navigate to the self DM
5. Click on 'Catgorize it' under the new expense
6. Select the new workspace

## Expected Result:
Hmm its not here page shouldn't be displayed momentarily

## Actual Result:
Hmmm it is not here page is displayed momentarily

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [ ] Android: mWeb Chrome
- [ ] iOS: Native
- [ ] iOS: mWeb Safari
- [ ] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

Add any screenshot/video evidence

</details> 

https://github.com/Expensify/App/assets/78819774/1dbb4eb9-ee69-44c4-9fb2-4eb7b9610203

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by lanitochka17** [comment]
@sonialiap  FYI I haven't added the External label as I wasn't 100% sure about this issue. Please take a look and add the label if you agree it's a bug and can be handled by external contributors

**Comment by sonialiap** [comment]
A polish issue that imo isn't super high value but since we already have a proposal, passing to external

**Comment by mananjadhav** [comment]
I think @ZhenjaHorbach's proposal looks good.

🎀 👀 🎀 C+ reviewed.

**Comment by mananjadhav** [comment]
PR is ready but being blocked by merge freeze. @Beamantor do you know where we can get this merged?

**Comment by Beamanator** [comment]
Yes we should be able to get this merged soon! probably in the next few days 🙏

**Comment by mananjadhav** [comment]
I've done the latest retest.

**Comment by mananjadhav** [comment]
@Beamanator  @sonialiap I would want to request to raise the payout for this one. It wasn't a simple bug fix. We ended up adding a new route, and did fix two issues in one PR. The scope of the implementation was a bit bigger than I anticipated.

@ZhenjaHorbach can chime in too if they agree/disagree.

**Comment by sonialiap** [comment]
@Beamanator I think you probably have a better grasp on the engineering work that went into this. Do you agree with a price increase?

**Comment by Beamanator** [comment]
I agree that a bump to $500 sounds fair 👍  Personally, I wouldn't think it needs more than that though - the PR isn't really huge and not overly complicated. Thoughts?

**Comment by sonialiap** [comment]
Cool, updating payments 

Payment summary:

- @mananjadhav $500 - please request in ND and complete [the checklist](https://github.com/Expensify/App/issues/40999#issuecomment-2152286026)
- @ZhenjaHorbach $500 - paid in upwork ✔️

**Comment by JmillsExpensify** [comment]
$500 approved for @mananjadhav

**Comment by sonialiap** [comment]
@mananjadhav please complete [the checklist](https://github.com/Expensify/App/issues/40999#issuecomment-2152286026)

**Comment by mananjadhav** [comment]
This was more like a new feature and I think it makes sense to add a regression test for this one.

1. Sign up with a new account
2. Create manual expense in Self DM
3. Create workspace
4. Navigate to the self DM
5. Click on 'Categorize it' under the new expense
6. Select the new workspace
7. Verify that the loader or list of elements are shown instead of Not here
8. Open workspace setting and open categories
9. Remove all items
10. Navigate to the self DM
11. Click on 'Categorize it'
12. Select the workspace with removed categories
13. Verify that the component is displayed indicating an empty list.
14. The screen should have an `Edit Categories` button, and clicking on the button should navigate you to add categories page.

**Comment by sonialiap** [comment]
Thanks, Manan! That looks good to me
